style.css

```diff
@@ -39,38 +40,40 @@
 	top: 0;
 	width: 4%;
 }
-.option-bar{
+.optionBar{
```

Comment:
`.option-bar`

Reply:
Then it shouldn't be slide__option-bar???